requests_mock/mocker.pyi

```diff
@@ -30,71 +30,73 @@ class MockerCore:
     def call_count(self) -> int: ...
     def register_uri(
       self,
-      method: str,
+      method: Union[str, Any],
+      url: Union[str, Pattern[str], Any],
```

Comment:
what's the point of union-ing with `Any`, does it give any hints over just using Any?

Reply:
AFAIK, no. Well, some tooling may go "hey, you might want to use str, but you can use Any", but mypy I don't think will care.